optaplanner-benchmark/src/test/java/org/optaplanner/benchmark/config/blueprint/SolverBenchmarkBluePrintConfigTest.java

```diff
@@ -0,0 +1,20 @@
+package org.optaplanner.benchmark.config.blueprint;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class SolverBenchmarkBluePrintConfigTest {
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @Test
+    public void throwIllegalArgumentException_WhenValidateOnUnassignedSolverBenchmarkBluePrintType() {
+        exceptionRule.expect(IllegalArgumentException.class);
+        exceptionRule.expectMessage("The solverBenchmarkBluePrint must have a solverBenchmarkBluePrintType (");
```

Comment:
instead of this, we use 
@Test (expected = IllegalArgumentException.class)

That might not test the exception message, but I am not a fan of hard coding the exception message in the test, for maintenance reasons.

Reply:
A couple of comments:
- @Rule annotation is no longer supported in jUnit5, thus I would rather avoid introducing more of them
- let's use AssertJ for checking exceptions as we already do in some tests below

Reply:
sorry for this one, thought I've removed all the @Rule usages, this one must've slipped through unnoticed